resources/views/cart/partials/products.blade.php

```diff
@@ -8,8 +8,7 @@
             <th class="text-end max-md:hidden">@lang('Subtotal')</th>
         </tr>
     </thead>
-    <tbody class="divide-y">
-        <tr v-for="(item, itemId, index) in cart.items" class="flex flex-wrap gap-y-5 py-5 md:table-row md:[&>*]:p-5">
+        <tr v-for="(item, index) in cart.items" v-if="cart.items" class="flex flex-wrap gap-y-5 py-5 md:table-row md:[&>*]:p-5">
```

Comment:
New tailwind feature you can use is `*:p-5`

Reply:
Wouldn't that select the current element as well as containing elements, while this selector only selects all child elements?